### ISSUE TITLE
Contribuisci anche tu! > Come Contribuire  (Fabrizio)

### ISSUE THREAD

**Issue by FrancescoUrbano** [hyperledger/fabric-docs-i18n#674]
Fix not working links

**Comment by FabrizioPe** [comment]
Thanks, I'll try and fix the links asap

**Comment by FabrizioPe** [comment]
@FrancescoUrbano 
It seems very strange that the links are broken, because I was very careful to leave them as they were.
In order to figure out what was wrong I looked at the CONTRIBUTING.rst file in the french directory:

fabric-docs-i18n/docs/locale/fr_FR/source, 

which is still an untraslated English version. The links have the exact same form as the ones in my file.
So, as far as I understand, the problem was already there when I forked the repo.
I'll investigate more anyway...
